soda/core/soda/scan.py

```diff
@@ -311,16 +312,10 @@ def execute(self) -> int:
                     from soda.sampler.soda_cloud_sampler import SodaCloudSampler
 
                     if isinstance(self._configuration.sampler, SodaCloudSampler):
-                        self._configuration.sampler = None
+                        self._configuration.sampler = DefaultSampler()
                 else:
-                    try:
-                        if self._configuration.soda_cloud.is_samples_disabled():
-                            self._configuration.sampler = None
-                    except Exception as e:
-                        self._logs.error(
-                            "Connecting to Soda Cloud failed.  Disabling Soda Cloud connection.", exception=e
-                        )
-                        self._configuration.soda_cloud = None
+                    if self._configuration.soda_cloud.is_samples_disabled():
+                        self._configuration.sampler = DefaultSampler()
```

Comment:
Why remove the exception handling?  

The reasoning behind it is that this is the first request to Soda Cloud.  If this first one fails, then we assume that all subsequent requests will fail as well.  So we remove Soda Cloud to prevent all subsequent errors logging.

Could be good to re-add this catch, no?